src/ucan.ts

```diff
@@ -152,33 +160,35 @@ export interface UCANOptions<
   nonce?: string
 
   facts?: Fact[]
-  proofs?: Array<Proof>
+  proofs?: Link[]
 }
 
 /**
- * Represents a {@link Link} to a UCAN (in either IPLD or JWT format) that serves as
- * proof for the capabilities claimed in another UCAN.
+ * Represents a UCAN {@link IPLDLink} in either IPLD or JWT format
+ * 
+ * @template Cap - {@link Capability}
+ * @template Alg - multicodec code corresponding to the hashing algorithm of the CID
  */
-export type Proof<
-  C extends Capability = Capability,
-  A extends number = number
-> = Link<Model<C>, typeof CBOR_CODE, A> | Link<JWT<C>, typeof RAW_CODE, A>
+export type Link<
+  Cap extends Capability = Capability,
+  Alg extends number = number
+> = IPLDLink<Model<Cap>, typeof CBOR_CODE, Alg, 1> | IPLDLink<JWT<Cap>,typeof RAW_CODE, Alg, 1>
```

Comment:
Nit: You could leave out version as `1` is a default.

```suggestion
> = IPLDLink<Model<Cap>, typeof CBOR_CODE, Alg> | IPLDLink<JWT<Cap>,typeof RAW_CODE, Alg>
```